src/utils/titleValidation.ts

```diff
@@ -0,0 +1,13 @@
+const titleValidation = (title: string) => {
+  try {
+    const filterTitle = JSON.parse(title)['title']
+      ? JSON.parse(title)['title']
+      : title;
+
+    return filterTitle;
+  } catch (error) {
+    return title;
```

Comment:
사소하긴 하지만 이 부분 이러면 parse 한번만 수행해도 되지 않을까라는 생각을 했습니다..
   
```suggestion
  const parseTitle = JSON.parse(title)[title] ; 
  return parseTitle ?? title 
```

Reply:
아 이 부분은 순수 string값을 parse 할시 바로 오류가 나와서 try{...}catch{...} 문으로 감싸야합니다ㅠㅠ
성래님 의견대로 한다면 오류때문에 동작을 안하게 됩니다ㅠㅠ